src/main/java/com/powsybl/openloadflow/equations/EquationTerm.java

```diff
@@ -178,11 +178,11 @@ static <V extends Enum<V> & Quantity, E extends Enum<E> & Quantity> EquationTerm
     double eval();
 
     /**
-     * Evaluate the equation term with an alternative state vector.
+     * Evaluate equation lhs of the equation term
      * @return value of the equation term
      */
-    default double eval(StateVector sv) {
-        throw new UnsupportedOperationException("Not implemented");
+    default double evalLhs() {
+        return eval() + (hasRhs() ? rhs() : 0);
```

Comment:
Should it be - rhs instead of + rhs ?
```suggestion
        return eval() - (hasRhs() ? rhs() : 0);
```

Reply:
You are right. Done.